src/Modal.js

```diff
@@ -74,6 +83,13 @@ class Modal extends Component {
     $(`#${this.modalID}`).modal('open');
   }
 
+  hideModal (e) {
+    if (e) e.preventDefault();
+    const { modalOptions = {} } = this.props;
+    $(`#${this.modalID}`).modal(modalOptions);
```

Comment:
Do we need to initialise the modal again or would `close` be enough

Reply:
We do not. I removed the call to $.modal and assert it isn't called an extra time when hiding the modal in the spec